shell/browser/javascript_environment.h

```diff
@@ -10,6 +10,7 @@
 #include "base/macros.h"
 #include "gin/public/isolate_holder.h"
 #include "uv.h"  // NOLINT(build/include_directory)
+#include "v8/include/v8-locker.h"
```

Comment:
Is including `v8/include/v8.h` no longer enough to get access to `v8::Locker`? That's surprising!

Reply:
@nornagon we don't include `v8.h` here - `gin/public/isolate_holder.h` only has `v8/include/v8-forward.h`

Reply:
ah, gotcha. thanks!